pyproject.toml

```diff
@@ -11,8 +11,8 @@ include = ["CHANGELOG.md"]
 [tool.poetry.dependencies]
 python = "^3.7.1"
 thrift = "^0.13.0"
-pyarrow = "^5.0.0"
 pandas = "^1.3.0"
```

Comment:
IMHO this library should drop support for python 3.7 & the update pandas to version `1.4.3`.

Reply:
Understandable. Because many Databricks users connect from older Pythons we prefer to maintain as much backwards compatibility as possible. For your use-case, is there any reason you cannot install a newer pandas? Or does `databricks-sql-connector` use pandas methods that were deprecated in newer versions?

Reply:
No -^ it isn't an issue for us. It is rather that many libraries have dropped support for python 3.7 & you may wish to update pandas to a newer version at some point which will mean dropping support for python 3.7.